### PR TITLE
[KYUUBI #2257] Replace replace conf vars with strings in HiveSQLEngine

### PR DESCRIPTION
**Pull request by zhaomin1423**
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Review by SteNicholas — APPROVED** [review]
LGTM. cc @yaooqinn

File: externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/HiveSQLEngine.scala [hunk 1]
```diff
@@ -77,10 +77,10 @@ object HiveSQLEngine extends Logging {
       (msConnUrl != null && msConnUrl.startsWith("jdbc:derby"))
     }
     if (isEmbeddedMetaStore) {
-      hiveConf.setBoolean("hive.metastore.schema.verification", false)
-      hiveConf.setBoolean("datanucleus.schema.autoCreateAll", true)
+      hiveConf.setBoolean(ConfVars.METASTORE_SCHEMA_VERIFICATION.varname, false)
+      hiveConf.setBoolean(ConfVars.METASTORE_AUTO_CREATE_ALL.varname, true)
       hiveConf.set(
-        "hive.metastore.warehouse.dir",
+        ConfVars.METASTOREWAREHOUSE.varname,
```
[the remark this answers is not in the document]
Comment (SteNicholas) [reply externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/HiveSQLEngine.scala:83]: @zhaomin1423, could you support these configurations in multiple version?

**Comment by yaooqinn** [comment]
actually，string representations are much more able compatible with multi versions

**Comment by yaooqinn** [comment]
we shall do the opposite to replace conf vars with strings

**Comment by zhaomin1423** [comment]
Thanks, I will close the pr later.

**Comment by yaooqinn** [comment]
> Thanks, I will close the pr later.

you can change  https://github.com/apache/incubator-kyuubi/pull/2259/files#diff-005f05920f3bf09d3ae2f59f1c92d8af0f9805aeb3c7b41f9a814ed0f3c59057L74-L75 instead

File: externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/HiveSQLEngine.scala (hunk 1 above)
[the remark this answers is not in the document]
Comment (zhaomin1423) [reply externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/HiveSQLEngine.scala]: Thanks, I have been replaced conf vars with strings, they are marked as deprecated in hive-3.1.3.

**Comment by zhaomin1423** [comment]
> > Thanks, I will close the pr later.
> 
> you can change https://github.com/apache/incubator-kyuubi/pull/2259/files#diff-005f05920f3bf09d3ae2f59f1c92d8af0f9805aeb3c7b41f9a814ed0f3c59057L74-L75 instead, and other places using ConfVars

Thanks, done.

**Review by ulysses-you — APPROVED** [review]

**Comment by yaooqinn** [comment]
thanks, merged to master
